docs/tutorials/03-on-build.md

```diff
@@ -0,0 +1,103 @@
+![buildah logo](../../logos/buildah-logo_large.png)
+
+# Buildah Tutorial 3 
+## Using ONBUILD in Buildah 
+
+The purpose of this tutorial is to demonstrate how Buildah can use a Dockerfile with the OnBuild command within it or how it can be used with the `buildah config` command.  The ONBUILD command stores a command in the meta data of the image which is then invoked whn the container is created.
```

Comment:
Note: The ONBUILD command does not change the content of the Image that is built by the Dockerfile containing it, only on container images that are created based off this image, using the FROM command.

Reply:
tweaked slightly and added.